tests/utils/misc/misc.go

```diff
@@ -21,8 +21,29 @@ import (
 	"time"
 )
 
+type vmstatus func(string) string
```

Comment:
1. Can we give this function a better name? "vmstatus" doesn't look like a function. 
2. Is it possible to use interface or pointers to support any type of functions? I'm not sure about this, just want to know if it's possible to extend this to a general solution that accepts any functions.